services/secret-service/doc/openapi.yaml

```diff
@@ -482,13 +646,14 @@ components:
     Owner:
       type: object
       required:
-        - id
-        - type
+        - entityId
```

Comment:
This change violates [API Spec Guidelines](https://github.com/openintegrationhub/openintegrationhub/blob/master/Guidelines/ServiceAPIGuidelines.md).

Reply:
We had an issue in the past when defining `id`, `type` in a schema. We'll check this one and align it to api spec.